pkg/cli/credential_login.go

```diff
@@ -89,3 +98,23 @@ func (a *CredentialLogin) Run(cmd *cobra.Command, args []string) error {
 	fmt.Println(cred.Name)
 	return nil
 }
+
+// validateCredentials takes a CredentialLogin and host string to validate
+// whether their combination is valid and will succeed login
+func validateCredentials(ctx context.Context, cred *CredentialLogin, host string) error {
```

Comment:
@ibuildthecloud need your input here. Do you want to do this validation client (cli) side or do you want it done server side?

Reply:
If we do decide on this, I have a change locally that I can easily push up to make it be server side.

Reply:
I would think server, an internal registry might be reachable by the cluster but not the client.

Reply:
yeah, server side, go for it

Reply:
Alright, the latest push moved this server side.